### ISSUE TITLE
Ability to have multiple SetErrorInterceptors

### ISSUE THREAD

**Issue by niemyjski** [mgonto/restangular#986]
I have multiple modules and each one might handle a different status code like 401 or 409. Would it be possible to support multiple error interceptors? https://github.com/mgonto/restangular/issues/488

Maybe it runs all of them and stops when one of them is handled.

**Comment by enapupe** [comment]
I've just got the same issue. That would be very good.

**Comment by grabbou** [comment]
This is already implemented and unit tests are passing, what's the issue you are having (code samples plz) ?

See those lines https://github.com/mgonto/restangular/blob/master/test/restangularSpec.js#L183-L211

**Comment by niemyjski** [comment]
I had multiple interceptors setup in different modules and the ordering of them really messed up things.  It's been a while (I sense reverted the code), but basically I always wanted them all to run, not stop after the first one returned a result.

**Comment by daviesgeek** [comment]
@niemyjski Any update? Is this still an issue?

**Comment by niemyjski** [comment]
Yes I do think so. I haven't tried for a long while as there hasn't been any updates to this lib

**Comment by daviesgeek** [comment]
Okay, would you mind providing a live example on Plunkr?
